openmmtools/multistate/multistatesampler.py

```diff
@@ -1309,45 +1309,73 @@ def _minimize_replica(self, replica_id, tolerance, max_iterations):
         thermodynamic_state_id = self._replica_thermodynamic_states[replica_id]
         thermodynamic_state = self._thermodynamic_states[thermodynamic_state_id]
         sampler_state = self._sampler_states[replica_id]
-
-        # Use the FIRE minimizer
-        integrator = FIREMinimizationIntegrator(tolerance=tolerance)
+        # NOTE: At this point, sampler_state.potential_energy is None. Initialize below.
+
+        logger.debug('Sampler state {}/{}:'.format(replica_id + 1, self.n_replicas))
+
+        # Temporarily disable the barostat during minimization by setting the
+        # pressure to None. (Otherwise, the minimizer will modify the box
+        # vectors and may cause instabilities which are very difficult to debug.)
+        pressure = thermodynamic_state.pressure
+        thermodynamic_state.pressure = None
+
+        # Use Gradient Descent first for numerical stability
+        integrator_grad = GradientDescentMinimizationIntegrator()
```

Comment:
The initial step size here is not guaranteed to be robust. I wonder if there's a way we can automatically select a more robust initial step size, or if this is not a critical issue.

Reply:
In some of my other work with gromacs, I use emstep 0.0001 (nanometers) which is 10 times smaller than the openmmtools default of 0.01 Angstroms. gromacs also has an adaptive step size, and I have some plots showing that starting with 0.001 Angstroms, the adaptive step size increases for the first ~25 iterations before stabilizing. In other words, you're only 'wasting' at most ~25 iterations, which isn't even a rounding error in the total runtime. Of course, those iterations are well spent if they can prevent minimization failures.